spec/models/document_spec.rb

```diff
@@ -57,8 +57,8 @@
     end
 
     it "should send out email-notifications" do
-      allow(valid_document).to receive(:recipients).and_return([user.mail])
-      allow(Notifier).to receive(:notify?).with(:document_added).and_return(true)
+      allow(valid_document).to receive(:recipients).and_return([user])
+      Setting.notified_events = Setting.notified_events << 'document_added'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.